LogMonitor/docs/README.md

```diff
@@ -0,0 +1,186 @@
+# Log Monitor Documentation
+
+**Contents:**
+
+- [ETW Monitoring](#etw-monitoring)
+- [Event Log](#event-log)
```

Comment:
link is broken: `#event-log-monitoring`?

Reply:
fixed